content/kubernetes/deployment/openshift/openshift-operatorhub.md

```diff
@@ -84,4 +84,7 @@ The **Installed Operators**->**Operator details** page shows the provided APIs:
 
 Use the YAML view to create a custom resource file or let OperatorHub generate the YAML file for you by specifying your configuration options in the form view.
 
+{{<note>}} In versions 6.4.2-4 and 6.4.2-5, REC creation fails when using the form view due to an error related to the cluster level LDAP. To avoid this, use the YAML view.
```

Comment:
```suggestion
{{<note>}} In versions 6.4.2-4 and 6.4.2-5, REC creation might fail when using the form view due to an error related to the cluster level LDAP. To avoid this, use the YAML view.
```